lifecycle/lifecycle-runtime-compose/gradle.properties

```diff
@@ -15,4 +15,4 @@
 #
 
 # TODO Determine redirection group based on package and remove explicit config
```

Comment:
Did we have some progress here?

Reply:
I didn't do anything for it.

After reading and thinking about it - it looks like determining it based on the package inside classes or based on the group either isn't possible or fragile, as different modules have different patterns.

We have to explicitly define mapping for redirection somewhere.

Reply:
I can remove this TODO if you agree

Reply:
This TODO is still valid to me.
Let's discuss offline, I have related changes in new adoption of buildSrc